src/interface.ts

```diff
@@ -85,8 +75,36 @@ export type AggregateVerifierParams = {
   verifier_id: string;
 };
 
+export interface Auth0UserInfo {
+  picture: string;
+  email: string;
+  name: string;
+  sub: string;
+  nickname: string;
+}
+
+export interface SingleFactorUserInfo extends Auth0UserInfo {
+  verifier: string;
+  verifierId: string;
+  typeOfLogin: string;
+}
+
 export interface SessionData {
   privKey?: string;
+  userInfo?: SingleFactorUserInfo;
+}
+
+export interface IWeb3Auth {
+  sessionId: string | null;
```

Comment:
no need to expose this to the dapp right.. use status variable like in no-modal sdk

Reply:
we need it for SFA react-native